packages/web/README.md

```diff
@@ -24,9 +24,25 @@ In the `@algo-builder/web` package we pass transaction [parameters](https://gith
 
 ### Example
 
-To use `web` package in your react app, first you need to create an instance of the `WebMode` class by passing `AlgoSigner` and the chain name.
+You can connect to `web` package in your react app by using different wallets.
 
-    const web = new WebMode(AlgoSigner, CHAIN_NAME);
+1.  ##### AlgoSigner:
+    Create an instance of the `WebMode` class by passing `AlgoSigner` and the chain name.
```

Comment:
I think we should link a template/example where this integration is used in a webapp for each wallet